src-built-in/components/authentication/app.jsx

```diff
@@ -0,0 +1,93 @@
+/*!
+* The authentication component is a React component that renders a user form to collect auth info and determine whether to give user access to the rest of finsemble
```

Comment:
Use caps for consistency.
```suggestion
* The authentication component is a React component that renders a user form to collect auth info and determine whether to give user access to the rest of Finsemble
```